grpc-protocol/src/main/java/com/linecorp/armeria/client/grpc/protocol/UnaryGrpcClient.java

```diff
@@ -205,32 +210,77 @@ public HttpResponse execute(ClientRequestContext ctx, HttpRequest req) {
                 private HttpData content = HttpData.empty();
                 @Nullable
                 private HttpHeaders trailers;
+                @Nullable
+                private Subscription subscription;
+                private boolean terminated;
```

Comment:
nit: How about renaming to `completed` which we have been using in other places.
https://github.com/line/armeria/blob/d3e25529570e340730aed7f83302e9b3fdf4c3ac/core/src/main/java/com/linecorp/armeria/common/stream/FixedStreamMessage.java#L208-L211
```suggestion
                private boolean completed;
```